test/unit/component/parser/json/BpmnJsonParser.sub.process.test.ts

```diff
@@ -78,12 +83,140 @@ describe('parse bpmn as json for sub-process', () => {
             bpmnElementName: 'sub-process name',
             bpmnElementKind: ShapeBpmnElementKind.SUB_PROCESS,
             bpmnElementMarkers: expectedBpmnElementMarkers,
-            bpmnElementIncomingIds: ['flow_in_1', 'flow_in_2'],
-            bpmnElementOutgoingIds: ['flow_out_1'],
             bounds: { x: 67, y: 23, width: 456, height: 123 },
           });
         },
       );
+
+      describe(`incoming/outgoing management for ${expandedKind} ${bpmnSubProcessKind} sub-process`, () => {
+        it.each`
+          title       | input         | expectedAttribute
+          ${'string'} | ${'incoming'} | ${'bpmnElementIncomingIds'}
+          ${'array'}  | ${'incoming'} | ${'bpmnElementIncomingIds'}
+          ${'string'} | ${'outgoing'} | ${'bpmnElementOutgoingIds'}
+          ${'array'}  | ${'outgoing'} | ${'bpmnElementOutgoingIds'}
+        `(
+          `should convert as Shape, when a process contains a ${expandedKind} ${bpmnSubProcessKind} sub-process with $input attribute as $title`,
```

Comment:
**nitpick:** the test name doesn't explicit that the incoming/outgoing are inferred. See remark in the "flow nodes" test file